src/Castle.Core.Tests/Core.Tests/Logging/LevelFilteredLoggerTests.cs

```diff
@@ -1014,11 +1014,11 @@ public void ErrorLevelInfoWithException()
 		[Test]
 		public void ErrorLevelWarnWithException()
 		{
-			logger.Level = LoggerLevel.Error;
+			logger.Level = LoggerLevel.Warn;
 			Exception exception = new Exception();
 
 			logger.Error("Test", exception);
-
+			
```

Comment:
I overlooked this whitespace-only change yesterday; sorry for that. Could you please revert it?